sdk/python/feast/infra/offline_stores/redshift.py

```diff
@@ -201,8 +196,6 @@ def __init__(
             config: Feast repo config
             full_feature_names: Whether to add the feature view prefixes to the feature names
             on_demand_feature_views: A list of on demand transforms to apply at retrieval time
-            drop_columns: Optionally a list of columns to drop before unloading to S3.
-                          This is a convenient field, since "SELECT ... EXCEPT col" isn't supported in Redshift.
```

Comment:
I will be honest with you but this concept of dropping columns is bad. If some SQL query engines do not support `SELECT * EXCEPT`, then let's not support it !

It's very weird to add this implementation outside of the SQL template. It makes things very different between BigQuery and other offline query engines

Thoughts about keeping it ?

Reply:
I can be missing something, so feel free to change my mind 🙂

Reply:
+1 to this change. Pretty sure it's just BQ that supports EXCEPT in this way

Reply:
it also is clearer to understand the query output